src/sentry/api/endpoints/organization_integration_issues.py

```diff
@@ -1,24 +1,33 @@
+from typing import Any
+
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.organization_integrations import RegionOrganizationIntegrationBaseEndpoint
 from sentry.integrations.mixins import IssueSyncMixin
 from sentry.services.hybrid_cloud.integration import integration_service
+from sentry.services.hybrid_cloud.organization import RpcUserOrganizationContext
 
 
 @region_silo_endpoint
 class OrganizationIntegrationIssuesEndpoint(RegionOrganizationIntegrationBaseEndpoint):
```

Comment:
Here is an example of an end endpoint I guess. It still doesn't seem to be public. There are definitely more. Should these end endpoints be public? If no, why not?

Reply:
No.  Integration endpoints, in general, are callbacks for partner integrations.  Anyone who is not that partner, for instance, won't have any of the secrets or values that back calling the endpoint.

For instance, this endpoint is called when Jira or other "issue" 3rd parties want to update the state of sentry using secrets that come from a customer's usages of that 3rd party.  In general, as a result, no integration endpoint is public.

Reply:
Makes sense. Thanks for the context.